### PR TITLE
Adds navigation controller wrapping overriding support to UIOperation #193

### PR DESCRIPTION
**Pull request by seancatkinson**
+ tests

### CODE REVIEW

**Comment by danthorpe** [comment]
Thanks! @seancatkinson! Very nice :+1:  

Let me know when you're done and I'll merge.

**Comment by seancatkinson** [comment]
The "wrapInNavigationController" parameter feels a bit wordy but I was struggling to think of anything more succinct. Unless you can think of something better I'm happy to merge if you're happy.

**Comment by danthorpe** [comment]
Yeah, I know what you mean. I'm happy to merge it for now, I might change it later if I can think of something a little more succinct.

**Comment by danthorpe** [comment]
Awesome! Thanks for this!
